using-timescaledb/ingesting-data.md

```diff
@@ -0,0 +1,23 @@
+# Ingesting data
+
+TimescaleDB can support standard SQL inserts. Read more about how to use
+SQL to write data into TimescaleDB in our [Writing Data][writing-data] section.
+
+Users often choose to leverage existing 3rd party tools to build data ingest
+pipelines. Any tools that work with PostgreSQL also work with TimescaleDB.
```

Comment:
"Users often choose to leverage existing 3rd party tools to build data ingest pipelines, especially in order to increase ingest rates by performing *batch* writes into TimescaleDB, as opposed to inserting data one row or metric at a time.   At a high-level, TimescaleDB looks just like PostgreSQL, so any tool that can read and/or write to PostgreSQL also work with TimescaleDB.

Below, we talk about some popular frameworks and systems used in conjunction with TimescaleDB."

Reply:
Done with some minor edits.